web/src/stream_edit.js

```diff
@@ -520,12 +521,17 @@ export function initialize() {
         });
         $("#show-sender").prop("checked", true);
 
-        new ClipboardJS("#copy_email_address_modal .dialog_submit_button", {
+        const clipboard = new ClipboardJS("#copy_email_address_modal .dialog_submit_button", {
             text() {
                 return address;
             },
         });
 
+        // Show a tippy tooltip when the stream email address copied
+        clipboard.on("success", (e) => {
+            show_copied_confirmation(e.trigger);
+        });
+
```

Comment:
The preview of this shown in your first GIF feels weird. Maybe we shouldn't close the modal?

Reply:
Ok, disabled close